fw/http_limits.c

```diff
@@ -1327,7 +1385,7 @@ static int
 frang_resp_fwd_process(TfwHttpResp *resp)
 {
 	int r;
-	FrangAcc *ra;
+	FrangAcc *ra = NULL;
```

Comment:
we always initialize `ra` at the below in line 1402, so no need to set it to NULL here